res/values/strings.xml

```diff
@@ -37,7 +37,6 @@
     <string name="one_moment">One moment…</string>
     <string name="done">Done</string>
     <string name="undo">Undo</string>
-    <!-- Translators: used eg. for the next view, could also be "continue" or so. as used in ios headers, the string should be as short as possible, though. -->
```

Comment:
why remove this hint?

Reply:
Because it was not necessary.

Reply:
i think it is helpful as "Next" can be translated to different words in different languages. the hint is shown eg. in transifex and other tools and gives some context